docs/pages/pmd/devdocs/pmd_devdocs_metrics_howto.md

```diff
@@ -0,0 +1,16 @@
+---
+title:  Using code metrics in custom rules
+tags: [customizing]
+summary: "PMD was recently enhanced with the ability to compute code metrics on Java code (the so-called Metrics 
+Framework). This framework provides developers with a straightforward interface to use code metrics in their rules, and 
+to extend the framework with their own custom metrics."
+last_updated: July 20, 2017
+sidebar: pmd_sidebar
+permalink: pmd_devdocs_metrics_howto.html
```

Comment:
I'm wondering, whether we really need this flat structure or whether we could reuse the folder structure? One argument was, that it's easier, if you link between the pages or include images, etc. You do not need to care about relative paths... But maybe, it would be better to have urls like https://pmd.github.io/pmd/devdocs/metrics_howto.html instead of https://pmd.github.io/pmd/pmd_devdocs_metrics_howto.html?